api/src/test/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImplTest.java

```diff
@@ -96,14 +104,41 @@ void modifyTermMatch(String testInput, String term, String expected) {
         .isEqualTo(expected);
   }
 
+  private static Stream<Arguments> getModifyTermMatchEndsWithParameters() {
+
+    return Stream.of(
+        // special chars are filtered by the UI-except ("\"", "+", "-", "*")
+        Arguments.of("Search term: ", "lung", "+lung*"),
+        Arguments.of("Search term: ", "+lung", "+lung*"),
+        Arguments.of("Search term: ", "lung cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung* cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung cancer*", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung* cancer*", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "+lung cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung +cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "+lung +cancer", "+lung*+cancer*"),
+        Arguments.of("Search term: ", "lung -cancer", "+lung*-cancer"),
+        Arguments.of("Search term: ", "+lung -cancer", "+lung*-cancer"),
+        Arguments.of("Search term: ", "lung* -cancer", "+lung*-cancer"),
+        Arguments.of("Search term: ", "\"lung cancer\"", "\"lung cancer\""),
```

Comment:
The result needs to be -> +"lung cancer"
```suggestion
        Arguments.of("Search term: ", "\"lung cancer\"", "+\"lung cancer\""),
```

Reply:
updated tests